airgap/k3d/zarf.yaml

```diff
@@ -47,3 +47,10 @@ components:
                   docker run --rm --entrypoint /bin/sh -v k3s-airgap-images:/dest -v $PWD:/src rancher/k3s:${ZARF_CONST_K3S_VERSION} -c "cp /src/k3s-airgap-images.tar.zst /dest"
                   rm k3d-proxy.tar k3d-tools.tar k3d-image.tar k3s-airgap-images.tar.zst
               description: Load the airgap images for k3d into Docker
+      onRemove:
+        after:
+          - cmd: |
+              # Check and remove k3s-airgap-images volume if it exists
+              if [ -n "$(docker volume ls -q --filter name=^k3s-airgap-images$)" ]; then
+                docker volume rm "k3s-airgap-images"
+              fi
```

Comment:
Probably just missing context, but this feels a bit weird when we don't have an onRemove deleting the cluster that is using this volume?

Reply:
Just for context, forgot to link this the other day - https://github.com/defenseunicorns/uds-k3d/issues/40#issuecomment-1949239136 . I think this has also changed with the introduction of the airgap flavor, but would want to carefully test here and make sure this is the best way to manage the removal/destroy. I don't recall the referenced jankiness, it may have been to do with the way Zarf makes cluster connections during destroys - not sure if things have also changed there.

TLDR: Should test carefully and validate if this is the best way to handle or if a clear document command(s) are better to do the removal.

Reply:
Pending this - will move this to draft until then: https://github.com/zarf-dev/zarf/pull/4267